engines/grim/lua_v1_graphics.cpp

```diff
@@ -509,6 +520,7 @@ void Lua_V1::SetGamma() {
 }
 
 void Lua_V1::Display() {
+	g_grim->drawCursor();
```

Comment:
this collide with remastered

Reply:
Last time I asked, I was told that remastered is not supported.

Reply:
yes, but it should not be executed for remaster

Reply:
I added an override in `Lua_Remastered`.

Reply:
add check for remastered, do not split